resolwe_bio/tests/processes/test_pca.py

```diff
@@ -58,6 +72,16 @@ def test_pca(self):
         self.assertAlmostEqualGeneric(test_json['components'], saved_json['components'])
         self.assertEqual(len(pca.process_warning), 0)
 
+        inputs = {
+            'exps': [
+                expression_noname_1.pk,
+                expression_noname_2.pk,
+            ],
+            'source': 'DICTYBASE',
+            'species': 'Dictyostelium discoideum',
+        }
+        pca = self.run_process('pca', inputs)
```

Comment:
Is the fact that process completes without error a sign of success or do you want to make some additional assertions? For example asserting there are no warnings, etc. I am fine with the way it is, but you know best weather this is bulletproof...